pkg/analytics/analytics.go

```diff
@@ -110,6 +121,7 @@ func NewRemoteAnalytics(appName string, options ...Option) (*remoteAnalytics, er
 		app:        appName,
 		url:        statsEndpt,
 		userID:     getUserID(),
+		machineID: getMachineID(),
```

Comment:
should we fold this into globalTags? (i was thinking of doing the same thing with userID)